File: conditional/notes.py
#Zoey Sosa, conditional notes python
name = input("what is your name?:\n")
#What puts something inside of the “if” statement?
if name == "LaRose":
    print("hi Mrs. LaRose")
else: #this happens if the codition is false
    print(f"hello {name}!")
#What do if statements do?
    #checks a condition and it it is true it will do a thing
if name == "LaRose": # <=- this is the condition 
    print("hi Mrs. LaRose") #<=this is what it does if true 

#What are boolean statements? 
if name == "LaRose": # <= this is the boolean statement. it is either true or false
    print("hi Mrs. LaRose")

#What do else statements do?
else: # if the boolean is false, the else statemnet happens 
    print(f"hello {name}!")

#What kind of statement do you use if you have more than 2 needed outcomes?
num = 2
#computers read top to bottom, check the least likely first 
if num == 0: #<= if always starts the conditional 
    print("There are none.")
elif num == 1: #everything inbetween should be elif 
    print("There is one.")
elif num <10:
    print("There is a few")
else : # <= else always ends the conditional 
    print("There are many")

#What do each of the different symbols mean in conditionals?

#< less than
#> greater than
#<= less then or equl to 
#>= greater then or equl to 
#== equal to 
#=== * dosent exsist in python 
#! not 

#What are the 3 logical operators?
if num <10 and num > 5: # And means that both booleans must be true 
    print("this is a big single digit number")

if num <10 or num > 5: # or means that one boolean must be true 
    print("this is a big single digit number")

elif not num < 10: # not changes to check if false
    print("this is a big single digit number")
#What are logical operators for?
    #Allows the code to handle more difficult quesittions . . . 
    #increases compelexity 
#What does a nested conditional statement do?
if num <10:
    if num  ==8: 
        print("this prints at 8")
    else:
        print("the number is less than 10")

#How do you write an if statement in C?
#How do you write else statements in C?
#How do you write elif/ else if statements in C?
#How do you write the 3 logical operators in C?

File: conditional/notes.c
//zoey sosa, conditional notes for C
#include <stdio.h>
#include <string.h> // needed to compare strings

char name[]="zoey";
int num;

int main(void){
//10. How do you write an if statement in C?
if(strcmp(name, "zoey")== 0){ // strcmp means string comparison when the outcome is 0 the string are the same 
    print("hello zoey, welcome to class\n");
//11. How do you write else statements in C?
}else{ 
    printf("hello %s, welcome to class.\n", name);
}

print("How many siblings do you have?\n");
scanf("%d", &num);
//12. How do you write elif/ else if statements in C?
if(num == 0){
    printf("you are a only child\n");
}else if (num <=2){
    printf("you have a couple siblings\n");
}else is (num <=5);{
    printf("you have a few siblings\n");
}
}

//13. How do you write the 3 logical operators in C?
//&& = and 
// ||= or
// ! = not
if (num == 7 || num == 13){
    printf("%d is an unlucky number\n", num);
}else if(num <10 && num >5){
    printf("%d is a large single digit number\n", num);
}else if (!num >10){
    if(num == 12){
        printf("that is a dozen!\n");
        }else{
            printf("%d is a big number\n", num);
    }else{
        printf("you type %d\n", num);
}
    return 0;
}